Lib/asyncio/runners.py

```diff
@@ -118,7 +120,14 @@ def run(self, coro, *, context=None):
                 events.set_event_loop(self._loop)
             return self._loop.run_until_complete(task)
         except exceptions.CancelledError:
-            if self._interrupt_count > 0 and task.uncancel() == 0:
+            uncancel = getattr(task, "uncancel", None)
+            if uncancel is None:
+                def uncancel():
+                    warnings._deprecated('missing asyncio.Task.uncancel',
```

Comment:
From a Python Steering Council discussion on the 3.11 release blockers: Lets not add a deprecation warning here. The decision to deprecate the old behavior can be made later.

Reply:
Agreed.

Reply:
Removed warning.